src/plone/restapi/tests/test_services_querystring.py

```diff
@@ -99,10 +99,26 @@ def test_endpoint_inlines_vocabularies(self):
             "published": {"title": "Published with accent \xe9 [published]"},
             "visible": {"title": "Public draft [visible]"},
         }
+
```

Comment:
With your refactoring of the tests, the 'expected_vocab_values' can already be updated with the additional token.